Tests/MapboxMapsTests/Style/StyleTransitionTests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+
+#if canImport(MapboxMaps)
+@testable import MapboxMaps
+#else
+@testable import MapboxMapsStyle
+#endif
```

Comment:
```suggestion
@testable import MapboxMaps
```

This can be simplified since we don't maintain the module structure anymore.